Header.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react'
+
+var backgroundImageCSS = {
+}
+
+class Header extends React.Component {
+  render() {
+    return (
+      <div className="jumbotron text-center eat-header-caption">
+        <div className="container">
+          <div className="row" >
+            <div className="col-md-12" >
```

Comment:
Just a bootstrap thing but if you have a col md 12 means that when its on a medium sized screen it should be a full row. That's also the default behavior. So it's kind of redundant